readme.md

```diff
@@ -57,45 +90,61 @@ testing
 }
 ```
 
-### Size Benefits
+#### This output represents a npm packages ready to be used without locally installing them!
+
+---
+
+## Comparisions
 
-One very significant upside of using deps using the `loader` and `import-map`. The dependencies from `test.js` loads
+Additionally, `@jspm/node-importmap-loader` can produce significant size benefits
+
+#### With regular npm install
 
 ```sh
-du -sh -I @jspm -I fs-extra -I node-fetch node_modules
+du -hs node_modules
 50M    node_modules
 ```
 
+#### With `@jspm/node-importmap-loader`
+
 ```sh
 du -sh .cache
 3.6M    .cache
 ```
 
----
+#### And it takes almost the same time to execute scripts!
```

Comment:
We unfortunately don't have metrics for this. Because the first time the scripts are executed. We do `fetch` calls. Which will slow down the very first starts in theory. So, please remove this. Maybe we can add once we have the metrics.